include/igraph_adjlist.h

```diff
@@ -81,8 +80,10 @@ typedef struct igraph_inclist_t {
 
 IGRAPH_EXPORT int igraph_inclist_init(const igraph_t *graph,
                                       igraph_inclist_t *il,
-                                      igraph_neimode_t mode);
+                                      igraph_neimode_t mode,
+                                      igraph_loops_t loops);
 IGRAPH_EXPORT int igraph_inclist_init_empty(igraph_inclist_t *il, igraph_integer_t n);
+IGRAPH_EXPORT igraph_integer_t igraph_inclist_size(const igraph_inclist_t *al);
```

Comment:
Add to CHANGELOG

Reply:
> If using `IGRAPH_LOOPS_TWICE` looks too ugly in the directed case, then let us write `IGRAPH_LOOPS` there in acknowledgement that it doesn't matter.

I'll do this and also swap the value of `IGRAPH_LOOPS_ONCE` and `IGRAPH_LOOPS_TWICE` as suggested.